front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/_shared/ProjectCarrousel/LightProjectCard.tsx

```diff
@@ -116,33 +115,35 @@ const LightProjectCard = ({ project, ml, mr, onKeyDown }: Props) => {
         </ProjectImageContainer>
       </Box>
       <Title variant="h4" mt="8px" mb="0px">
-        {title}
+        {truncate(title, 50)}
       </Title>
-      <Box mt="8px">
-        <Icon
-          name="clock-circle"
-          fill={colors.blue500}
-          height="16px"
-          mr="4px"
-          ml="-4px"
-          transform="translate(0,-1)"
-        />
-        <Text color="blue500" fontWeight="bold" display="inline" m="0">
-          {end_at ? (
-            <PhaseTimeLeft currentPhaseEndsAt={end_at} />
-          ) : (
-            <>{formatMessage(messages.noEndDate)}</>
-          )}
-        </Text>
-        <Text mt="4px" mb="0px" color="textSecondary" fontWeight="bold">
-          {getCTAMessage({
-            phase: phase.data,
-            actionDescriptors: project.attributes.action_descriptors,
-            localize,
-            formatMessage,
-          })}
-        </Text>
-      </Box>
+      {phase && (
+        <Box mt="8px">
```

Comment:
Only show this bit if there is a current phase. This means that if there is no current phase, we don't show 'Time left' and the call to action. In a future PR we will tweak this by e.g. showing how many days since the project ended or, if it hasn't started yet, how many days till start. But that requires some more BE work